src-vue/src/plugins/store.ts

```diff
@@ -479,6 +488,42 @@ async function _checkForFlightCoreUpdates(state: FlightCoreStore) {
     }
 }
 
+async function flightcoreUpdateCheck() {
+    const update = await check();
+    if (!update?.available) {
+        console.log("No update available");
+    } else if (update?.available) {
+        console.log("Update available!", update.version, update.body);
+        const accepted = await ask(
+        `Update to ${update.version} is available!\n\nRelease notes: ${update.body}`,
```

Comment:
Actually, we should also translate this one.

Reply:
Addressed in 655f8f32b04df0a10afc708d806530863c67febb